bitnami/minio/templates/provisioning-job.yaml

```diff
@@ -96,8 +96,13 @@ spec:
               mc admin policy add {{ $minioAlias }} {{ $policy.name }} /etc/ilm/policy-{{ $policy.name }}.json;
               {{- end }}
 
-              {{- range $user := .Values.provisioning.users }}
-              mc admin user add {{ $minioAlias }} {{ $user.username }} {{ $user.password }};
+              {{- range $idx, $user := .Values.provisioning.users }}
+              {{- $userPassword := default "" $user.password }}
```

Comment:
TBH, I am not sure if the empty default value is problematic. It would only be used in case neither inline password nor secret reference is specified. Maybe in that case the release should throw an error? Happy for opinions on that.

Reply:
Possible options instead of empty value:

- `hasKey` check to ensure password field is set if new approach is not used
- create random password
- let `helm install` fail with proper error message (as mentioned above)